[2015-05-29T18:23:45] ERROR: (<class 'UnboundLocalError'>, UnboundLocalError("local variable 'data' referenced before assignment",), <traceback object at 0x0209EF80>)
[2015-05-29T18:23:45] ERROR: [('E:\\SublimeProjects\\TheToyboxDev\\TheToybox\\Python\\StreamtoVLC\\stream.py', 888, '<module>', 'main()'), ('E:\\SublimeProjects\\TheToyboxDev\\TheToybox\\Python\\StreamtoVLC\\stream.py', 873, 'main', 'debug([str(sys.exc_info()), str(traceback.extract_stack())],1)')]
